Signal/src/ViewControllers/MediaGalleryViewController.swift

```diff
@@ -39,13 +39,13 @@ public class MediaGalleryItem: Equatable, Hashable {
     // MARK: Equatable
 
     public static func == (lhs: MediaGalleryItem, rhs: MediaGalleryItem) -> Bool {
-        return lhs.message.uniqueId == rhs.message.uniqueId
+        return lhs.attachmentStream.uniqueId == rhs.attachmentStream.uniqueId
     }
 
     // MARK: Hashable
 
     public var hashValue: Int {
-        return message.hashValue
+        return attachmentStream.hashValue
```

Comment:
`attachmentStream.uniqueId.hashValue`?

Reply:
Yeah, I think that'd make more sense. Updated!